experimental/dds/tree2/package.json

```diff
@@ -33,7 +33,7 @@
 		"format": "npm run prettier:fix",
 		"lint": "npm run prettier && npm run check:release-tags && npm run eslint",
 		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
-		"postpack": "tar -cf ./experimental-tree2.test-files.tar ./src/test ./dist/test",
+		"postpack": "tar -cf ./tree.test-files.tar ./src/test ./dist/test",
```

Comment:
Of note, this doesn't conflict with the test files from legacy tree, which is still in the @fluid-experimental scope and names its test files tarfile `experimental-tree.test-files.tar` accordingly.